pkg/expr/identity.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package expr
+
+import (
+	"fmt"
+
+	"istio.io/mixer/pkg/attribute"
+)
+
+// identity is an evaluator that expects mapExpression to be a key in the attribute bag. It does no evaluation, only lookup.
+type identity struct{}
+
+// NewIdentityEvaluator returns an evaluator that performs no evaluations; instead it uses the provided mapExpression
```

Comment:
Can this just be merged in the normal expression evaluator?

@mandarjog As we discussed, an expression should be either a lone attribute name, a litteral value, or a complex expression that produces an attribute value.